src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -94,11 +98,17 @@ constexpr size_t minimum_number_of_points(
 template <Basis basis, Quadrature quadrature>
 constexpr size_t minimum_number_of_points =
     detail::minimum_number_of_points(basis, quadrature);
+
 /*!
  * \brief Maximum number of allowed collocation points.
+ *
+ * Since we cache points using compile time lookup we are limited to 10000
+ * points for FiniteDifference. However, realistically 25 points is sufficient
+ * for efficient parallelization and for DG-subcell.
  */
-template <Basis>
-constexpr size_t maximum_number_of_points = 12;
+template <Basis basis>
+constexpr size_t maximum_number_of_points =
+    basis == Basis::FiniteDifference ? 25 : 12;
```

Comment:
It could break subcell, though you'd get an `ERROR` (or `ASSERT`, not sure off the top of my head). The reason I say could is because I very much doubt we would run DG at beyond 12th order with hydro, and I sort of think if someone is trying to do shock capturing at twelfth order they've earned some error messages :)

I don't know of a way to specialize it and the call the "unspecialized" version. I've also had compilers give me a lot of issues with specializing variable templates (GCC blows up a lot, though I don't remember exactly which versions), which is why this uses a ternary to begin with.

Reply:
Okay, sounds good. Leave as is then 👍